src/Generators/MigrationGenerator.php

```diff
@@ -156,26 +156,44 @@ protected function buildDefinition(Model $model)
         return trim($definition);
     }
 
-    protected function buildPivotTableDefinition(array $segments, $dataType = 'foreignId')
+    protected function buildPivotTableDefinition(array $segments, $dataType = 'unsignedBigInteger')
     {
         $definition = '';
 
         foreach ($segments as $segment) {
             $column = Str::lower($segment);
-            $foreignColumn = 'id';
-            $foreignTable = Str::plural($column);
-            $localeColumn = Str::singular($column) . '_' . $foreignColumn;
+            $references = 'id';
+            $on = Str::plural($column);
+            $foreign = Str::singular($column) . '_' . $references;
 
-            if (config('blueprint.use_constraints')) {
-                if ($this->isLaravel7orNewer()) {
-                    $definition .= self::INDENT . '$table->' . $dataType . "('{$localeColumn}')->constrained('{$foreignTable}', '{$foreignColumn}');" . PHP_EOL;
-                } else {
-                    $definition .= self::INDENT . '$table->unsignedBigInteger' . "('{$localeColumn}');" . PHP_EOL;
-                    $definition .= self::INDENT . '$table->foreign' . "('{$localeColumn}')->references('{$foreignColumn}')->on('{$foreignTable}');" . PHP_EOL;
+            $definition .= $this->buildConstraint($foreign, $on, $references, $dataType);
+        }
+
+        return trim($definition);
+    }
+
+    protected function buildConstraint(string $foreign, string $on, string $references = 'id', string $dataType = 'unsignedBigInteger', ?string $onDelete = null)
```

Comment:
What are the other options to `onDelete`?

Reply:
Default MySQL constraints are `['cascade', 'set null', 'restrict']` and I assumed Laravel had support for all the options, but now apparently they have them through other way, will adjust this.

Reply:
What do you mean "now". Like in Laravel 7 they changed?

Reply:
[This](https://stackoverflow.com/questions/39518162/available-actions-for-onupdate-ondelete-in-laravel-5-x) stack-overflow issue from (5.3) addressed this, I quickly assumed this was the way to go since it seemed rather simple. But doing further research the Laravel (7.x) documentation also addresses this but very subtle; [Laravel Docs](https://laravel.com/docs/7.x/migrations#foreign-key-constraints) 

> You may also specify the desired action for the "on delete" and "on update" properties of the constraint:
```php
$table->foreignId('user_id')
      ->constrained()
      ->onDelete('cascade');
```